k8s/aat/software-engineering/draft-store/draft-store-service.yaml

```
@@ -0,0 +1,39 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: draft-store-service
+  namespace: software-engineering
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:latest*
+spec:
+  releaseName: draft-store-service
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
+    name: draft-store-service
+    version: 0.0.2
+  values:
+    java:
+      applicationInsightsInstrumentKey: "a2e12f21-e77a-4b9f-91f3-6e7b3867892e"
+      image: hmcts.azurecr.io/hmcts/draft-store-service:latest
+      ingressHost: draft-store-service.service.core-compute-aat.internal
+      environment:
+        DRAFT_STORE_DB_CONN_OPTIONS: ?sslmode=require
+        DRAFT_STORE_DB_HOST: rpe-draft-store-aat.postgres.database.azure.com
+        DRAFT_STORE_DB_NAME: draftstore
+        DRAFT_STORE_DB_PORT: 5432
+        DRAFT_STORE_DB_USER_NAME: draftstore@rpe-draft-store-aat
+        RUN_DB_MIGRATION_ON_STARTUP: false
```

```suggestion
        RUN_DB_MIGRATION_ON_STARTUP: 'false'
```

same with booleans

don't you want this true?

---

I saw the value in app services environment and it was set to false in AAT.

---

hmm actually is probably fine to be false
but it should be quoted anyway